Content.Shared/Inventory/VirtualItem/SharedVirtualItemSystem.cs

```diff
@@ -178,7 +200,7 @@ public void DeleteInSlotMatching(EntityUid user, EntityUid matching, string? slo
     /// </summary>
     /// <param name="blockingEnt">The entity we will make a virtual entity copy of</param>
     /// <param name="user">The entity that we want to insert the virtual entity</param>
-    public bool TrySpawnVirtualItem(EntityUid blockingEnt, EntityUid user, [NotNullWhen(true)] out EntityUid? virtualItem)
+    public bool TrySpawnVirtualItem(EntityUid blockingEnt, EntityUid user, [NotNullWhen(true)] out EntityUid? virtualItem, bool dropOthers = false)
```

Comment:
Should this be removed?

Reply:
Fixed in [`e35366a` (#27975)](https://github.com/space-wizards/space-station-14/pull/27975/commits/e35366a8167e5d19091ab4cf9aebe916c5580d58)